Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -52,10 +46,26 @@ static DWRITE_TEXT_ALIGNMENT __CoreTextAlignmentToDwrite(CTTextAlignment alignme
     }
 }
 
+static DWRITE_WORD_WRAPPING __CTLineBreakModeToDWrite(CTLineBreakMode lineBreakMode) {
```

Comment:
__ is file-private, _ is framework-"private" (not intended to be used by general consumers but exported nonetheless)

---
In reply to: [90742638](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90742638) [](ancestors = 90742638)